thehive4py/api.py

```diff
@@ -835,6 +855,8 @@ def get_alert(self, alert_id):
             AlertException: An error occured during alert update
         """
         req = self.url + "/api/alert/{}".format(alert_id)
+        if get_similar_cases:
+            req = req + "?similarity=1"
```

Comment:
This is valid, but I would have used a param arg to the `request.get` call

Reply:
Do you mean something like:

~~~
params = {
  "similarity": int(get_similar_cases)
}
return requests.get(req, params = params, proxies=self.proxies, auth=self.auth, verify=self.cert)
~~~